contribs/src/main/java/com/netflix/conductor/contribs/queue/amqp/AMQPObservableQueue.java

```diff
@@ -366,27 +359,23 @@ public AMQPObservableQueue build(final boolean useExchange, final String queueUR
     }
 
     private Channel getOrCreateChannel() {
-        if (!isConnOpened) {
-            open();
-        }
-        // Return the existing channel if it's still opened
-        if (channel != null && isChanOpened) {
+        // Return the existing channel if it was created
+        if (channel != null) {
```

Comment:
Can we also check it the channel is not closed? An existing channel which is already closed will also cause issues